### ISSUE TITLE
The differences between `env_batch` and `batch_size`

### ISSUE THREAD

**Issue by Vandermode** [hzwer/ICCV2019-LearningToPaint#28]
Hi, I dived into the code of your paper and I'm confused of the two variables `env_batch` and `batch_size`, which seems to be the same according to your implementation.

Could you give me some hints to help me figure it out? Thank you very much

**Comment by hzwer** [comment]
env_batch refers to how many strokes are generated each time when interacting with the environment. batchsize is the number of samples per gradient descent.

**Comment by Vandermode** [comment]
Thank you for your response!  but I'm still wondering about the benefit of such a design choice. Why not   directly set env_batch = batchsize?

**Comment by hzwer** [comment]
Because there is no discriminator when inference, GPU memory usage is different. Moreover, the proportion of time for training and environment interaction can be adjusted.
